test/pipes/basename-test.js

```diff
@@ -0,0 +1,30 @@
+const buster = require('buster')
+const pipe = require('../../lib/pipes/basename')
+
+buster.testCase('Pipe: basename', {
+  'should remove the basename from file': function () {
```

Comment:
Da gibts doch nen test dazu:
'if there are more extension then only the last should be removed'
Oder was genau meinst du damit?